common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -296,9 +296,11 @@ def copy_or_rename_transcript(new_name, old_name, item, delete_old=False, user=N
 def get_html5_ids(html5_sources):
     """
     Helper method to parse out an HTML5 source into the ideas
-    NOTE: This assumes that '/' are not in the filename
+    NOTE: This assumes that '/' are not in the filename.
+    Slices each id by 100, restricting too long strings as video names.
     """
-    html5_ids = [x.split('/')[-1].rsplit('.', 1)[0] for x in html5_sources]
+    html5_ids = [x.split('/')[-1].rsplit('.', 1)[0][:100] for x in html5_sources]
```

Comment:
Currently there are two places where video name is extracted, one from JS at front end for operations like check/upload transcripts. One is here when someone hits save in editor for example after adding a new URL resource(also after uploading transcript). This new URL resource is supposed to have a transcript instance, so this method is used for extracting name for that transcript.